src/main/java/lotto/type/Rank.java

```diff
@@ -0,0 +1,74 @@
+package lotto.type;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public enum Rank {
+
+    FIRST(1, 6, 200000000),
+    SECOND(2,5, 30000000),
+    THIRD(3,5, 20000000),
+    FOURTH(4, 4, 50000),
+    FIFTH(5, 3, 10000),
+    MISS(6, 0, 0);
+
+    private int ranking;
+    private int countOfMatch;
+    private int winningMoney;
+
+    Rank(final int ranking, final int countOfMatch, final int winningMoney){
```

Comment:
```suggestion
    Rank(final int ranking, final int countOfMatch, final Money winningMoney){
```

Money 객체를 만드셨으니 Rank에서도 Money객체로 들고 있는 것은 어떨까요?